go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kedacore/keda
 
-go 1.13
```

Comment:
This is a regression that I by accident brought by migrating to operator-sdk 1.0.0 https://github.com/kedacore/keda/pull/1007.

We are already on 1.14

Reply:
If we don't go with 1.15, I'll open another PR to fix the version to the correct one (1.14)

Reply:
going to 1.15 sounds good to me.